app/helpers.php

```diff
@@ -74,3 +74,36 @@ function getPreference()
 {
     return App\Models\Settings\Setting::all()->first();
 }
+
+function slugify($str) {
```

Comment:
there's already a laravel helper that automatically does this so no need to create a new helper. just use `str_slug('String')`

Reply:
Son of abiriga. mine is more robust and named properly 😛 . will look into its source code.